components/restapi/core/blocks.go

```diff
@@ -61,3 +64,67 @@ func blockIssuance(_ echo.Context) (*apimodels.IssuanceBlockHeaderResponse, erro
 
 	return resp, nil
 }
+
+func sendBlock(c echo.Context) (*apimodels.BlockCreatedResponse, error) {
+	mimeType, err := httpserver.GetRequestContentType(c, httpserver.MIMEApplicationVendorIOTASerializerV2, echo.MIMEApplicationJSON)
+	if err != nil {
+		return nil, ierrors.Wrapf(httpserver.ErrInvalidParameter, "invalid block, error: %w", err)
+	}
+
+	var iotaBlock = &iotago.ProtocolBlock{}
+
+	if c.Request().Body == nil {
+		// bad request
+		return nil, ierrors.Wrap(httpserver.ErrInvalidParameter, "invalid block, error: request body missing")
+	}
+
+	bytes, err := io.ReadAll(c.Request().Body)
+	if err != nil {
+		return nil, ierrors.Wrapf(httpserver.ErrInvalidParameter, "invalid block, error: %w", err)
+	}
+
+	switch mimeType {
+	case echo.MIMEApplicationJSON:
+		// Do not validate here, the parents might need to be set
+		if err := deps.Protocol.CurrentAPI().JSONDecode(bytes, iotaBlock); err != nil {
```

Comment:
Add `serix.WithValidation()` and remove comment, since we are now expecting a fully valid block here